proto/osmosis/poolmanager/v1beta1/genesis.proto

```diff
@@ -89,6 +89,14 @@ message TakerFeeParams {
   string community_pool_denom_to_swap_non_whitelisted_assets_to = 5
       [ (gogoproto.moretags) =
             "yaml:\"community_pool_denom_to_swap_non_whitelisted_assets_to\"" ];
+
+  // bypass_whitelist is a list of addresses that are
+  // allowed to bypass the taker fee when performing a swap
+  // (i.e. swap without paying the taker fee).
+  // It is intended to be used for integrators who meet qualifying factors
+  // that are approved by governance.
+  repeated string bypass_whitelist = 6
+      [ (gogoproto.moretags) = "yaml:\"bypass_whitelist\"" ];
```

Comment:
and lets change this as well to imply reduced taker fee

Reply:
Done